CMakeLists.txt

```diff
@@ -77,6 +78,20 @@ find_package(FFTW3 REQUIRED)
 include_directories(${FFTW3_INCLUDES})
 
 ########################################################################
+#Process GROMACS settings
+########################################################################
+
+if(WITH_XTC)
+find_package(GROMACS CONFIG)
+include_directories(${GROMACS_INCLUDE_DIRS})
+  if(GROMACS_FOUND)
+    add_definitions( -DHAS_GROMACS )
+  else(GROMACS_FOUND)
+    set(WITH_XTC OFF)
+  endif(GROMACS_FOUND)
+endif(WITH_XTC)
```

Comment:
I guess, we could simplify it to:
```cmake
if(WITH_XTC)
￼  find_package(GROMACS CONFIG REQUIRED)
￼  include_directories(${GROMACS_INCLUDE_DIRS})
  add_definitions( -DHAS_GROMACS )
endif()
```
to enforce gromacs if (explicitly) enabled.